egs/callhome_diarization/v1/diarization/VB_resegmentation.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python
```

Comment:
Does this script work with python3? I believe that we've decided to accept the dependency on python3. So, if possible, I think we should replace python with python3 at the top of the script.